### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by YannickB** [OCA/vertical-community#11]
Hello everyone,

Following a discussion I had with lucode, I think we need to place the documentation within the repository. This way all contributions to the doc will be controlled by the regular PR process and contributors will be tracked by github stats.

I am still trying to find if we will use .md files or sphinx like in https://github.com/OCA/connector-magento/tree/8.0/__unported__/magentoerpconnect/doc. The documentation need to be integrated with a third party site like readthedocs to be easily read by newcomers.

**Comment by lucode** [comment]
I would favor .md files in the first place. First it shows the contend without clicking on the file. Second I can't find any simple editor for sphinx. I suggest to use sphinx for the code documentation and .md files more for the admin documentation. That would some of the content of my README.md could transferred into .rst files. That should also help noncoders to contribute to the documentation.

**Comment by YannickB** [comment]
I agree, markdown is best since I want noncoders to write the documentation.

I see them tomorrow to have their opinion on it. I already created the readthedocs page https://readthedocs.org/projects/vertical-community/, do you know how we shall place the files so it can be read by readthedocs?

**Comment by lucode** [comment]
The german language works now but not the menu items. Do I have to consider amething special with them.

**Comment by lucode** [comment]
Hi everybody,

I still don't fully understand how the mail_holocracy module works.
I can follow the module description (see below) but I couldn't figure how what the specific function should use for.

> Mail.group (Discussion Group) is now a recursive model. You can't subscribe to a parent group but each followers of a  group is automatically subscribed to his parent group
> A group can now be a normal group, a circle or a role
> In a circle, you can define permissions for children groups
> A group can now be linked to a partner, you can easily create it from the group

Another Question that's related to the messaging system is: how I can allow messaging between users. As far I now portal (share) users can send messages under themselves and regular also, But mixing does not work.  If I as a staff member want to communicate with the community I have to have two profiles one as regular and one as portal users. Are there any other ways?

**Comment by florent-valeureux** [comment]
Hi, I will try to clarify holacracy module. I will bring the theory and an example. For the theory:
- Discussion Group are only for talk matters without any specific rights
- Circle Group can have specific wallet and several rights spread between roles
- Roles Group are child of Circle Group and inherit or not rights from them.
- Members who followed a group will inherit his rights

I will take a company example, here CA-Technology. Within the module CA-Technology is a circle which represent a company with several rôles and circles inside. This main circle own a wallet for all the company. Wallet control is owned by the board rôle .

![dye7ev5 1](https://cloud.githubusercontent.com/assets/9918969/5425710/e7d73746-8322-11e4-8ce4-ae9688f14cfb.png)

*Designer circle group people involved in design.
*Board role have several rights it can control wallet, invitation and group management. 
*Engineering circle group people involved in technical matters. It have a specific wallet which is controlled by treasury role. Please not that this circle have his own circles and roles.

Member Aurélie follow engineering circle, design circle and board role. That means she follow project and discussion from engineering and design. But she also can control CA-Technology wallet as a member of Board role. Thus during transactions she can use wallet for CA-Technology. 

Circles Group goal is to spread leadership and responsibility to roles.

**Comment by lucode** [comment]
Hi florent,
Thank you very much for you elaborate introduction. This module does provide much more then I hoped for. I will make some tests om my testserver to fully comprehend your concept.

**Comment by YannickB** [comment]
Thank you florent for taking the time to explain, it's true the holacracy module is not the easiest module to understand.

That said, please don't hack the topic, this one is about how we gonna build the documentation. To me, we will place the documentation in markdown format under the docs folder. I just made a commit to illustrate this https://github.com/OCA/vertical-community/commit/25c039fdcb01ba2ffbbd95538187a445f2bf9da7.
Any contributions on the documentation is more than welcome, we already had some discussions about the verticalisation use in the issues here, it'd be good to move the answers under the documentation. Our modules are complex, we need the documentation to attract more people.

**Comment by lucode** [comment]
I will hopefully find some time next week to improve some parts of the documentation.

Lucas 

Von unterwegs mit K-9 Mail gesendet.
Sent from a mobile device.
